server/plugin/plg_editor_onlyoffice/index.go

```diff
@@ -236,6 +237,14 @@ func IframeContentHandler(ctx App, res http.ResponseWriter, req *http.Request) {
 		}
 		return ""
 	}()
+	lang = func() string {
+		lang := req.Header.Get("Accept-Language")
+		if lang == "" {
+			return "en"
+		}
+
+		return lang[0:2]
```

Comment:
This will potentially create runtime errors when lang isn't the expected size, eg: https://play.golang.org/p/9c5DiECSFjK